relayer/pkg/chainlink/ocr2/client_test.go

```diff
@@ -15,12 +17,12 @@ import (
 
 func TestOCR2Client(t *testing.T) {
```

Comment:
This test the way it is currently setup (depends on real infra) should be moved to `integration-tests` module.

Reply:
resolving this comment as it has been turned into an integration test :)